pypdf/_writer.py

```diff
@@ -2187,6 +2187,7 @@ def clean_forms(
 
             clean(content, images, forms)
             page.replace_contents(content)
+        return
```

Comment:
How is this a valid return value for the given type hints?

Reply:
This is an unstable fix and should use the correct return parameter as two lists.